### ISSUE TITLE
Settings list component - Create quo2 component

### ISSUE THREAD

**Issue by jo-mut** [status-im/status-react#15665]
Create the quo2 component Setting lists as per designs:
https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?node-id=1650%3A31097&t=0fFyi5RetW3JBgFI-1

Should be implemented according to best practices
https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md

Include component spec tests
https://github.com/status-im/status-mobile/blob/develop/doc/component-tests-overview.md

Communicate with design team before starting to make sure there is no unknown requirements

Please liase with @J-Son89 with regards of the API.

Only the first 6 variations are in scope for this task, but the API should accommodate to be extended to eventually accept all the options.

Blur is out of scope.

**Comment by rfahernandez** [comment]
FYI I'll be working on this issue
